notebooks/spacescope_port.ipynb

```diff
@@ -0,0 +1,7447 @@
+{
```

Comment:
Great job with this comparison notebook 👏 👏 
This makes the reviewer so much more confident about the changes 😄